src/ui/pages/Signin/Signin.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import LogNavigation from "../../components/AppNavigation/LogNavigation"
+import { createUserAcount } from "../../../infra/Authentication/Authentication";
+import "./Signin.css"
+
+import fondConnexion from "../../assets/fond-connexion.jpg"
+
+export default function Signin() {
+
+    const [errorMessage, setErrorMessage] = useState('');
+    const [userInfo, setUserInfo] = useState({
+        firstName: "",
+        lastName: "",
+        email: "",
+        password: ""
+    })
+
+    const handleChange = (e) => {
+        const value = e.target.value;
+        setUserInfo({...userInfo, [e.target.id]: value});
+    }
+
+    const createAccount = (e) => {
+        e.preventDefault();
+        for(let value in userInfo) {
```

Comment:
question test : tu connais une alternative au for(... in ...) ? :)

Reply:
For .. in .. pour itérer dans un objet ! Si je dis pas de bêtises, j'ai déjà fais un Object.keys(objet).map() .. tu penses à ça ?